server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -455,4 +513,31 @@ export class TenantManager {
         const accessInfo = JSON.parse(this.secretManager.decryptSecret(encryptedAccessInfo));
         return accessInfo;
     }
+
+    private decryptCachedKeys(cachedKey: string) {
+        const keys = JSON.parse(cachedKey);
+        return keys.key2 === "" ? {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: "",
+            } : {
+                key1: this.secretManager.decryptSecret(keys.key1),
+                key2: this.secretManager.decryptSecret(keys.key2),
+            };
+    }
+
+    private async getCacheKey(tenantId: string): Promise<string> {
+        return this.cache.get(`tenantKeys:${tenantId}`);
```

Comment:
From the definition, `this.cache` could be undefined. We should use `this.cache?.get...`. Same as below

Reply:
updated to this.cache?